psi4/driver/driver.py

```diff
@@ -499,6 +500,11 @@ def energy(name, **kwargs):
     """
     kwargs = p4util.kwargs_lower(kwargs)
 
+    # Bounce to MDI if mdi kwarg
+    use_mdi = kwargs.pop('mdi', False)
+    if use_mdi:
+        return mdi_run(name, **kwargs)
```

Comment:
Thanks! I'll rebase the distributed driver branch once this is in and see how it breaks :-)